ElectronClient/app.js

```diff
@@ -1178,7 +1181,8 @@ class Application extends BaseApplication {
 		for (const itemId of ['copy', 'paste', 'cut', 'selectAll', 'bold', 'italic', 'link', 'code', 'insertDateTime', 'commandStartExternalEditing', 'showLocalSearch']) {
 			const menuItem = Menu.getApplicationMenu().getMenuItemById(`edit:${itemId}`);
 			if (!menuItem) continue;
-			menuItem.enabled = !!note && note.markup_language === MarkupToHtml.MARKUP_LANGUAGE_MARKDOWN;
+			const isHtmlNote = !!note && note.markup_language === MarkupToHtml.MARKUP_LANGUAGE_MARKDOWN;
+			!isHtmlNote || viewer === 'viewer' ?  menuItem.enabled = false : menuItem.enabled = true;
```

Comment:
This is like writing `true ? x = false : x = true;` or you know, you could just write `x = false`. This is the same here, you could simply write `menuItem.enabled = !isHtmlNote && viewer !== 'viewer'`.

Reply:
I just tried it, it doesn’t work as expected, it breaks the expected functionality. I think this is the simplest and most readable way to solve the issue and maintain its expectated behavior.

Reply:
`menuItem.enabled = isHtmlNote && viewer !== 'viewer';`

Reply:
ah, now I see what the problem is. We don't want the items to be enabled if it is an HTML note or if it the viewer. The problem here is that `isHtmlNote` is false when it's an HTML note (and true when it's a Markdown note), so Runo you have it backward.

Essentially you wrote this: `isHtmlNote = note.type === 'markdown'` - doesn't that tell you that it's in fact a Markdown note, and not an HTML one? So full condition should be `const isHtmlNote = !!note && note.markup_language === MarkupToHtml.MARKUP_LANGUAGE_HTML;`

Reply:
And I insist on getting rid of `true ? x = false : x = true;`

Reply:
Perfect!

> it doesn’t work as expected, it breaks the expected functionality

@Runa-saduwa whenever you think this, it means it's time to look closer to find out why!

Reply:
> And I insist on getting rid of true ? x = false : x = true;

i apologize, I have figured it out. I’ve gotten rid this

Reply:
@mic704b you’re very correct. Thank you for that, I was a bit hasty in trying to solve the issue that brought about the PR

Reply:
All good, now make sure to read Laurent's comment above closely, and adjust the code as described if it makes sense.

Laurent said:
> Essentially you wrote this: `isHtmlNote = note.type === 'markdown'` - doesn't that tell you that it's in fact a Markdown note, and not an HTML one? So full condition should be `const isHtmlNote = !!note && note.markup_language === MarkupToHtml.MARKUP_LANGUAGE_HTML;`

Reply:
I’ve made the adjustments.

If this isn’t okay, I still have another way to do this, I’ve looked closely at the code and I’ve seen different ways to achieve what we want.